metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -206,7 +205,8 @@ class MetalsLanguageServer(
         )
       }
       new InitializeResult(capabilities)
-    }.logError("initialize").asJava
+    }.logError("initialize")
+    timed("initialize")(result).asJava
```

Comment:
Emacs is blocked during the `initialize` request so I delayed all the slow steps to run after the `initialized` notification, this greatly helped speed up startup time on Emacs (I haven't hit the same problem in other editors). We will log from now on how long the `initialize` request takes in order to prevent further regressions, it should only take a few milliseconds to wire together a few components.